### PR TITLE
Add completion params

### PR DESCRIPTION
**Pull request by EniasCailliau**
This PR adds most of OpenAI's completion params with the exception of `logit_bias`, `logprobs` and `stream`. 

Motivation:

* We don't support streaming completions so we can't support `stream` for now.  
* We don't expose `logprobs` in our Generation Tags. Don't think it's useful to fold that in now.  
* Logic_bias looks like an expert feature requiring us to support maps in our config template. Probably not worth it for now. 

Quick additions: 
- [x] Upgraded to Steamship 2.3.6
- [x] Added integration test

### CODE REVIEW

File: steamship.json [hunk 1]
```diff
@@ -31,6 +31,41 @@
...
+      "description": "How many completions to generate for each prompt.",
+      "default": 1
+    },
+    "echo": {
+      "type": "boolean",
+      "description": "Echo back the prompt in addition to the completion",
+      "default": false
+    },
+    "stop": {
+      "type": "string",
```
Comment (douglas-reid) [inline steamship.json:51]: with this being a `string` but the config being List[str], should we clarify how the list should be passed in somewhere?

File: test/test_integration.py [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+    return f"test_{''.join(random.choice(letters) for _ in range(10))}".lower()  # noqa: S311
+
+
+@pytest.fixture
+def plugin_instance(steamship: Steamship) -> PluginInstance:
+    """Instantiate a plugin instance."""
+    plugin_instance = steamship.use_plugin(
+        plugin_handle=BLOCKIFIER_HANDLE,
+        instance_handle=random_name(),
+        config=json.load(Path("config.json").open()),
```
Comment (douglas-reid) [inline test/test_integration.py:36]: does `config.json` exist ?

File: test/test_integration.py [hunk 3]
```diff
@@ -0,0 +1,64 @@
...
+    """
+    letters = string.digits + string.ascii_letters
+    return f"test_{''.join(random.choice(letters) for _ in range(10))}".lower()  # noqa: S311
+
+
+@pytest.fixture
+def plugin_instance(steamship: Steamship) -> PluginInstance:
+    """Instantiate a plugin instance."""
+    plugin_instance = steamship.use_plugin(
+        plugin_handle=BLOCKIFIER_HANDLE,
```
Comment (douglas-reid) [inline test/test_integration.py]: should this be `BLOCKIFIER_HANDLE` ?

File: steamship.json (hunk 1 above)
Comment (dkolas) [reply steamship.json:51]: Presumably in the description of the param

**Review by dkolas — APPROVED** [review]
Looks good to me.

**Pull request merged**